CSS/style.css

```diff
@@ -115,8 +116,40 @@ footer {
   margin-left: 10px;
```

Comment:
You can omit lines 111 & 112 (the float and margin for .social icons li) without really affecting the layout.